hazelcast/src/main/java/com/hazelcast/map/impl/record/AbstractRecord.java

```diff
@@ -28,17 +28,14 @@
 
     protected Data key;
     protected long version;
-    /**
-     * evictionCriteriaNumber may be used for LRU or LFU eviction depending on configuration.
-     */
-    protected long evictionCriteriaNumber;
     protected long ttl;
+    protected long creationTime;
+
     protected volatile long lastAccessTime;
     protected volatile long lastUpdateTime;
-    protected long creationTime;
+    protected volatile long hits;
```

Comment:
I have asked, because I don't know is there any concurrent mutating access for map record.